frontend/src/bundles/notifications/notifications-api.ts

```diff
@@ -20,7 +20,9 @@ class NotificationsApi extends BaseHttpApi {
         super({ path: ApiPath.NOTIFICATIONS, baseUrl, http, storage });
     }
 
-    public async fetchNotifications(): Promise<Array<NotificationResponseDto>> {
+    public async fetchNotifications(): Promise<{
+        items: NotificationResponseDto[];
+    }> {
```

Comment:
```suggestion
    public async fetchNotifications(): Promise<{
        items: Paged<NotificationResponseDto>;
    }> {
```
Response shape still seems to be wrong, double-check, but I believe that shape should be like this.

Reply:
Looks cleaner! https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/pull/454/commits/c66bf7077220ec349e1aa89e915e8000e38c297b